gradle.properties

```diff
@@ -1,11 +1,3 @@
-//  -------------------------------------------------------------------------
-//  Copyright (C) 2021 BMW AG
-//  -------------------------------------------------------------------------
-//  This Source Code Form is subject to the terms of the Mozilla Public
-//  License, v. 2.0. If a copy of the MPL was not distributed with this
-//  file, You can obtain one at https://mozilla.org/MPL/2.0/.
-//  -------------------------------------------------------------------------
-
```

Comment:
why?

Reply:
Android Studio doesn't like the header and reports syntax issues. It's a configuration file, so it doesn't strictly need the header

Reply:
yes, because a comment in this file starts with "#"